BaseTools/set_vsprefix_envs.bat

```diff
@@ -34,10 +34,18 @@ if defined VS140COMNTOOLS (
     set "VS2015_PREFIX=%VS140COMNTOOLS:~0,-14%"
   )
   if not defined WINSDK81_PREFIX (
-    set "WINSDK81_PREFIX=c:\Program Files\Windows Kits\8.1\bin\"
+    if exist "%ProgramFiles(x86)%\Windows Kits\8.1\bin" (

```

Comment:
WINSDK81_PREFIX should first check "%ProgramFiles%\Windows Kits\8.1\bin", then check "%ProgramFiles(x86)%\Windows Kits\8.1\bin"

Reply:
Thanks Liming,

I am OK with that.
(just wondering that is there possible to install both Win SDK 8.1 for both x86 version and also x64 version in one system?)

If based on your change, does it would be committed into trunk?
or I have another action need to take?

Sincerely,
Wilson (BIngo)

Reply:
This comment is for this patch. You just need to update this patch. After you update this patch, I will approve it.

Reply:
Thank Liming,

I commit & push the update.
Please help to confirm.

Sincerely,
Wilson (Bingo)

Reply:
Please combine two commits to single one, then push it.